pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/ExtensibleLoadManagerImpl.java

```diff
@@ -398,6 +425,50 @@ private boolean isInternalTopic(String topic) {
                 || topic.startsWith(TOP_BUNDLES_LOAD_DATA_STORE_TOPIC);
     }
 
+    private void playLeader() {
+        log.info("This broker:{} is the leader now.", pulsar.getLookupServiceAddress());
+        serviceUnitStateChannel.scheduleOwnershipMonitor();
+        this.pulsar.getLoadManagerExecutor().execute(() -> {
+            try {
+                loadStoreInitWaiter.await();
+            } catch (InterruptedException e) {
```

Comment:
After being interrupted, do we still need to start the table view?

Reply:
Optimistically, yes.

If the load store has been initialized, then it will successfully start table view.

If not, in the worst case, the table view start will throw an exception.

Reply:
In fact, I think we can retry when interrupted.

Added `waitForLoadStoreInit()`